exploring-ship-logbooks/word_count.py

```diff
@@ -0,0 +1,52 @@
+import pandas as pd
```

Comment:
#### I think pd is necessary, because the input variable "data" is actually a pd DataFrame.